### ISSUE TITLE
Add https://searx.baczek.me

### ISSUE THREAD

**Issue by Wint3rmute** [searxng/searx-instances#161]
### Requirements

- [X] It is my instance.
- [X] I give the right to `check.searx.space` to check my instance (every 3 hours for the response times, every 24 hours for the other tests).
- [X] I acknowledge that managing a public instance is not an easy task and require spending time to keep the instance in good health. E.g. look after your instance by using a monitoring system.
- [X] I guarantee to keep an uptime per month of my instance at **minimum 90%**. Please ask for a removal of your instance if there is a planned long downtime or notify us [here](https://github.com/searxng/searx-instances/discussions/categories/report-instance-downtime) for a short downtime.
- [X] I do not track the users of my instance with any analytics or tracking software.
- [X] I won't try to manipulate the ranking of my instance in a way that give an unfair advantage over the other public instances in the list. (e.g. caching requests for searx.space server)

### Bot protection

Yes, I have enabled the limiter plugin built in SearXNG.

### Source code URL

Vanilla https://github.com/searxng/searxng-docker.git

**Comment by Wint3rmute** [comment]
Just to make sure, the README at https://github.com/searxng/searxng-docker.git says the following

>If you add a new instance, allow the IPs of check.searx.space to access your instance.

If I'm hosting a SearxNG instance which is accessible by anyone (without any IP/location filtering), just as a regular web page, `check.searx.space` will be able to access it, right?

**Comment by tiekoetter** [comment]
> If I'm hosting a SearxNG instance which is accessible by anyone (without any IP/location filtering), just as a regular web page, `check.searx.space` will be able to access it without requiring any configuration modifications, right?

Yes this is correct.

**Comment by unixfox** [comment]
Hello @Wint3rmute,

This is a friendly reminder that your provider OVH offer IPv6, it would be great if you could add IPv6 to your instance.

It may be not difficult at all, you may already have IPv6 connectivity on your server, all you need is to add a record AAAA in your DNS server.

Here is the documentation here: https://support.us.ovhcloud.com/hc/en-us/articles/4406638994067-How-to-Configure-IPv6-on-a-VPS

The future of the internet will thank you, see [why here](https://old.reddit.com/r/explainlikeimfive/comments/1k45y7/eli5_what_is_ipv6_and_why_is_it_so_important/).

**Comment by Wint3rmute** [comment]
Hey @unixfox,

Thanks for the heads-up! My Searx instance seems to be passing the ready.chair6.net test now :)
![image](https://github.com/searxng/searx-instances/assets/19196352/232c0c36-a4d0-4cd9-b8a8-b624dd2f9715)

By the way, I've noticed that my instance's TLS score has changed to `E,?` on searx.space:
![image](https://github.com/searxng/searx-instances/assets/19196352/37587969-b9c2-4b00-b378-279d4058830f)

At the same time, [cryptcheck](https://cryptcheck.fr/https/searx.baczek.me) is telling me that my score is A+
![image](https://github.com/searxng/searx-instances/assets/19196352/14e33bfe-85f5-49b7-b435-3e564d228c1c)

Do you maybe have an idea what's going on here?

**Comment by unixfox** [comment]
@Wint3rmute last final warning, your instance do not have the parameter `public_instance` enabled to true: https://searx.baczek.me/config

**Comment by Wint3rmute** [comment]
Sorry, will fix today in the evening

**Comment by unixfox** [comment]
Your instance is outdated, please update it and keep it up to date.

**Comment by Wint3rmute** [comment]
Hey, thanks for the heads-up, I've updated the instance.

**Comment by unixfox** [comment]
your instance has been down a lot lately. why is that?
